src/validation.h

```diff
@@ -74,6 +74,7 @@ static const unsigned int DEFAULT_LIMITFREERELAY = 30;
 /** The maximum size for transactions we're willing to relay/mine */
 static const unsigned int MAX_STANDARD_TX_SIZE = 100000;
 static const unsigned int MAX_ZEROCOIN_TX_SIZE = 150000;
+static const unsigned int MAX_STANDARD_SHIELDED_TX_SIZE = MAX_TX_SIZE_AFTER_SAPLING;
```

Comment:
This sets the maximum  size for "standard" shielded txes, to the maximum available by consensus.
In other words, there would not be any non-standard (due to its size) shielded transaction.
Not sure we really want to set it this high.
But we can discuss the proper value of this constant in another PR.

Reply:
sounds good